pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "wheel", "calver"]
+requires = ["setuptools>=43.0.0", "wheel>=0.32.0", "calver"]
```

Comment:
This is unnecessary, since pip will always get the latest version, and there's currently no way to change that behavior in any way.

Reply:
Thanks.  Is this guaranteed for all frontends?  I'll revert this if there's an strong opinion (e.g. from you :smile:) against it.

Reply:
AFAIK, all frontends just use pip eventually.

Reply:
I don't think this is necessary to resolve https://github.com/pypa/trove-classifiers/issues/38.

Reply:
I'll imply two weak opinions as a strong one and revert this to avoid discussion about something that doesn't affect use case we are aware of :smile: